getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -3,93 +3,122 @@
+which are required to use C# in Godot.
 
-.. note:: For instructions on installing older versions of Mono on Linux, see `this page <http://www.mono-project.com/docs/getting-started/install/linux/#accessing-older-releases>`_.
-        Older versions of Mono for macOS and Windows can be found `here <https://download.mono-project.com/archive/>`_.
+.. note:: To download Mono on macOS, use the "Stable Channel" link
+          from the `Mono Downloads Page <http://www.mono-project.com/download/>`_.
+          The Visual Studio channel is an earlier version of Mono and
+          will not work.
 
-Additionally, your Godot version must have Mono support enabled, so ensure you download the **Mono version** of Godot.
+Additionally, your Godot version must have Mono support enabled,
+so make sure to download the **Mono version** of Godot.
+If you are building Godot from source, make sure to follow the steps to
+enable Mono support in your build as outlined in the
+:ref:`doc_compiling_with_mono` page.
 
-If you are building Godot from source, make sure to follow the steps to include Mono support in your build outlined on the :ref:`doc_compiling_with_mono` page.
-
-In summary, you must have installed 1) the correct Mono SDK version for your Godot version, and 2) The Mono version of Godot.
+In summary, you must have installed the correct Mono SDK version for your
+Godot version **and** the Mono-enabled version of Godot.
 
 Configuring an external editor
 ------------------------------
 
-While Godot does have its own scripting editor, its support for C# is kept
-minimal, and it's recommended that you use an external IDE or editor, such as
-Microsoft Visual Studio Code, or MonoDevelop, which provide auto-completion,
-debugging and other features useful when working with C#.
-To set it up, in Godot click on ``Editor``, then ``Editor Settings``. Scroll
-down to the bottom, to the ``Mono`` settings. Under Mono click on ``Editor``,
-and on that page choose your external editor of choice.
+While Godot has its own script editor, its support for C# is kept
+minimal; it's recommended that you use an external IDE or editor
+instead, such as `Visual Studio Code <https://code.visualstudio.com/>`_
+or MonoDevelop. These provide auto-completion, debugging and other features
+useful when working with C#. To set it up in Godot, click on
+**Editor → Editor Settings** and scroll down to the bottom to the
+**Mono** settings. Under **Mono**, click on **Editor** then choose
+your external editor of choice.
 
-.. note:: If you are using Visual Studio Code, ensure you download and install the .NET tools extension. Without this, Godot will crash when trying to create a new C# project or edit a C# script.
+.. note:: If you are using Visual Studio Code, ensure you download and install
+          the .NET Tools extension. Without this, Godot will crash when trying
+          to create a new C# project or edit a C# script.
```

Comment:
I don't know of any .NET Tools extension for VSCode. The recommended C# extension if this: https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp Also Godot shoudn't crash because of this.